scripts/build/companion_libs/220-ncurses.sh

```diff
@@ -49,7 +49,7 @@ do_ncurses_for_host() {
     # (i.e. in simple cross or native): ncurses may not be needed for
     # host, but we still need them on build to produce 'tic'.
     case "${CT_TOOLCHAIN_TYPE}" in
-        native|cross)   return 0;;
+        native)   return 0;;
```

Comment:
I've lost my debug data but if I recall correctly, in case of cross gdb, CC="${CC_for_gdb}" and LD="${LD_for_gdb}" in scripts/build/debug/300-gdb.sh expands to include library search and include paths which points to the host sysroot. Since only the build sysroot include ncurses static it does not find it and so TUI has indirectly been disabled for a long time. I assumed host sysroot is the correct one to populate in case of building host tools. Or, we need to fix so that it points to the build sysroot.